### PR TITLE
pd: make some configuration more clear

### PR DESCRIPTION
**Pull request by rleungx**
### What is changed, added or deleted? (Required)

make `low-space-ratio` and `high-space-ratio` more clear

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by disksing — APPROVED** [review]

File: pd-configuration-file.md [hunk 1]
```diff
@@ -250,14 +250,14 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
 
 ### `high-space-ratio`
 
-+ 设置 store 空间充裕的阈值。此配置仅在 `region-score-formula-version = v1` 时生效。
++ 设置 store 空间充裕的阈值。当节点的空间占用比例小于指定值时，PD 调度时会忽略剩余空间这个指标，主要针对实际数据量进行均衡。此配置仅在 `region-score-formula-version = v1` 时生效。
```
Comment (TomShawn) [inline pd-configuration-file.md:253]: + `指定值`是指什么呢？
```suggestion
+ 设置 store 空间充裕的阈值。当节点的空间占用比例小于指定值时，PD 调度时会忽略剩余空间这个指标，主要根据实际数据量进行均衡。此配置仅在 `region-score-formula-version = v1` 时生效。
```

File: pd-configuration-file.md [hunk 2]
```diff
@@ -250,14 +250,14 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
...
 + 默认：0.7
 + 最小值：大于 0
 + 最大值：小于 1
 
 ### `low-space-ratio`
 
-+ 设置 store 空间不足的阈值。
++ 设置 store 空间不足的阈值。当节点的空间占用比例超过指定值时，PD 会尽可能避免往对应节点迁移数据，同时主要针对剩余空间大小进行调度，避免对应节点磁盘空间被耗尽。
```
Comment (TomShawn) [inline pd-configuration-file.md:260]: `指定值`是指这个阈值吗？
```suggestion
+ 设置 store 空间不足的阈值。当节点的空间占用比例超过指定值时，PD 会尽可能避免往对应节点迁移数据，同时主要针对剩余空间大小进行调度，避免对应节点磁盘空间被耗尽。
```

Comment (rleungx) [reply pd-configuration-file.md:260]: 是的

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**